apps/web/docs/RATE_LIMITING_FLOW.md

```
@@ -0,0 +1,295 @@
+# Rate Limiting Flow Diagram
+
+## Request Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)